src/ui/public/vislib/lib/types/pie.js

```diff
@@ -0,0 +1,13 @@
+import _ from 'lodash';
+
+export default function PieConfig(Private) {
```

Comment:
so yea, the purpose of this functions under lib/types is to correctly convert data passed from kibana (renderbot) to the new config .... for pie chart it doesnt do much, as i havent changed pie charts.